packages/grid/_modules_/grid/hooks/features/column-reorder/useColumnReorder.tsx

```diff
@@ -147,6 +145,15 @@ export const useColumnReorder = (columnsRef: React.RefObject<HTMLDivElement>, ap
     [apiRef, logger],
   );
 
+  const colReorderApi: ColumnReorderApi = {
+    onColCellDragStart: handleDragStart,
+    onColHeaderDragOver: handleColumnHeaderDragOver,
```

Comment:
What's the diff between colCell and colHeader?

Reply:
The cell is what is inside `column-header-item` and the header is what's in the `column-header`.